HLTrigger/Configuration/python/CustomConfigs.py

```diff
@@ -31,6 +31,8 @@ def Base(process):
 #        process.GlobalTag.connect   = 'frontier://FrontierProd/CMS_CONDITIONS'
 #        process.GlobalTag.pfnPrefix = cms.untracked.string('frontier://Frontie#rProd/')
 #        
+    process.GlobalTag.snapshotTime = cms.string("9999-12-31 23:59:59.000")
```

Comment:
No: this is for the updated L1 menus versions - we can't wait for the GT PRs,
and some L1T menus will never make a GT (eg, the special LowPU L1T menus).